### DISCUSSION TITLE
committed to paying

### DISCUSSION THREAD

**Discussion by Solidario736** [community/community#73180] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

General

### Body

Good afternoon. If I pay now, how long am I committed to paying in the future. Thank you.

**Comment by mdavic0** [comment]
Good afternoon! The duration of your commitment depends on the specific terms and conditions of the payment you're making. Without more information about the context and the service or product you're paying for, it's challenging to provide a precise answer. I recommend checking the documentation or reaching out to the provider's customer support for clarification on the terms of your payment and any future commitments. They should be able to provide you with the relevant information. Thank you

**Comment by ghostinhershell** [comment]
Hey there!  👋

Thanks for posting in the GitHub Community, @Solidario736! We're happy you're here. You are more likely to get a useful response if you are posting your question in the applicable category. The Accessibility category is a place for our community to discuss and provide feedback on the digital accessibility of GitHub products. Digital accessibility means that GitHub tools, and technologies, are designed and developed so that people with disabilities can use them.

With that being said, can you please add more details to your question so that someone in the community can provide you with an appropriate answer? As you can see [by searching "billing" in our Docs](https://docs.github.com/en/search?query=billing), different GitHub products can be billed in different ways. 

I've gone ahead and moved this to the correct category for you. Good luck!
